src/PositionVelocityTime.jl

```diff
@@ -6,9 +6,11 @@ module PositionVelocityTime
     export  calc_PVT,
             is_sat_healthy_and_decodable,
             PVTSolution,
+            sat_position_rotation, 
             sat_position_ECI_2_ECEF, 
             sat_position_ECEF,
             user_position,
+            null_PVT,
```

Comment:
Rather than creating a extra `null_PVT` function, I would create an empty constructor for `PVTSolution`. Something like:
```
function PVTSolution()
    PVTSolution(ECEF([0,0,0]), 0, nothing, nothing, nothing, nothing, nothing, 0, nothing, nothing, nothing)
end
```
See also comment below. With this change the function `null_PVT` can be removed.